engine/src/test/java/io/zeebe/engine/processor/workflow/multiinstance/MultiInstanceActivityTest.java

```diff
@@ -589,13 +589,6 @@ public void shouldSetOutputElementVariable() {
     completeJobs(workflowInstanceKey, INPUT_COLLECTION.size());
 
     // then
-    assertThat(
-            RecordingExporter.variableRecords(VariableIntent.CREATED)
-                .withName(OUTPUT_ELEMENT_VARIABLE)
-                .limit(INPUT_COLLECTION.size()))
-        .extracting(r -> r.getValue().getValue())
-        .containsOnly("null");
-
```

Comment:
I think we can keep this check now after dealing with the `outputElement` variable name.